lib/libutee/include/tee_api_defines.h

```diff
@@ -449,9 +449,14 @@
  *      return TEE_ERROR_BAD_PARAMETERS;
  *  }
  */
-#define TEE_PARAM_TYPES(t0,t1,t2,t3) \
-   ((t0) | ((t1) << 4) | ((t2) << 8) | ((t3) << 12))
-
+#ifdef CFG_TEE_BENCHMARK
```

Comment:
Sure, but there are a lot of code inside optee linux kmod and optee core, that are really bothered about the correct value for param_types including the fifth TEE_PARAM_TYPE_MEMREF_INOUT.

There is lot of calls of TEE_PARAM_TYPE_GET(param->types, n); (check my for param cleanup [PR](https://github.com/OP-TEE/optee_os/pull/1154/commits/68540524672021166c8121b764d69c12e5c4a22d), you can find a lot of usage of this macro inside loops, just after changes for TEE_NUM_PARAMS etc.)